Additional fluid properties documentation & tests metadata (#24585)
idaholab/moose: modules/fluid_properties/doc/content/source/auxkernels/StagnationPressureAux.md

# StagnationPressureAux

!syntax description /AuxKernels/StagnationPressureAux

> Stagnation pressure is the pressure the fluid would obtain if brought to rest without loss of mechanical energy. It is

Reviewer: ```suggestion
Stagnation pressure, or *total* pressure, is the pressure the fluid would obtain if brought to rest isentropically. It is
```
